packages/astro/src/core/build/generate.ts

```diff
@@ -468,7 +468,11 @@ async function generatePath(
 
 	const outFolder = getOutFolder(config, pathname, route);
 	const outFile = getOutFile(config, outFolder, pathname, route);
-	route.distURL = outFile;
+	if (route.distURL) {
```

Comment:
i wonder if we can remove `distURL`? Do you know what it is used by @bholmesdev, I recall you adding this a long time back.

Reply:
Cloudflare uses it

Reply:
Wow, I honestly don't remember adding that. If Cloudflare uses it, that's reason enough. Can't think of other use cases